config/initializers/console.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module PublishConsole
+  def start
+    show_warning_message_about_environments
+    super
+  end
+
+  def show_warning_message_about_environments
+    if Settings.environment.name == 'beta'
```

Comment:
I think our check for production should be `Rails.env.production_aks?`, but the linter complains. What do you think?

`W: Rails/UnknownEnv: Unknown environment production_aks. Did you mean production?`

Reply:
Like this: https://github.com/DFE-Digital/publish-teacher-training/pull/4020/files

Reply:
Ah I posted my response before your second comment came through, will update! Thanks Avin